etc/languages.tsv

```diff
@@ -1,80 +1,80 @@
-Kabanae	Kabanae	Mondé	xxx	kbn	kbn	-60.519885	-6.333723	
+Kaapor	Ka'apor	Tupi-Guarani	urb	urb	urub1250	-46.2204	-2.46455	balee1994footprints,73670,ribeiro2015marcaccao,caldas2009proposta,kakumasu1988dicionario
+Kabanae	Kabanae	Mondé		kbn		-60.519885	-6.333723	
 Kaiowa	Kaiowa	Tupi-Guarani	kgk	kgk	kaiw1246	-54.6815	-24.589	553686,cardoso2008aspectos
 Kamajura	Kamajura	Tupi-Guarani	kay	kay	kama1373	-53.4651	-12.2661	35108,154298,29263,143531,28622,39576,53208,14073
-Kampe	Kampe	Tuparí	kmp	kmp	kmp	-61.193643	-12.790872	
+Kampe	Kampe	Tuparí	kmp			-61.193643	-12.790872	
 Karitiana	Karitiana	Arikem	ktn	ktn	kari1311	-64.1522	-9.37937	552373,158748,134597,38969,163773,52509,storto2019kin
 Karo	Karo	Ramarana-Purubora	arr	arr	karo1305	-61.5691	-10.3342	46651,gabas2007dicionario,gabas2001grammar
 Kayabi	Kayabi	Tupi-Guarani	kyz	kyz	kaya1329	-55.5479	-11.2747	dobson2005aspectos,faria2004ordem,51787,143577,27389,7774,307132
 Kepkiriwat	Kepkiriwat	Tuparí	kpn	kpn	kepk1241	-61.25	-11.66	313075,loukotka1968classification
 Kuruaya	Kuruaya	Munduruku	kyr	kyr	kuru1309	-52.2303	-3.09561	85060,319563,23568,469113,da2008fonologia,costa2016fonologia,picancco2020fonologia
-Tupí Austral	Tupí Austral	Tupi-Guarani	tpʷ	tpʷ	tupi1274	-46.38	-23.95	leite2013lingua,551239,martius_2009
+TupiAustral	Tupí Austral	Tupi-Guarani	tpʷ	tpʷ	tupi1274	-46.38	-23.95	leite2013lingua,551239,martius_2009
 Makurap	Makurap	Tuparí	mpu	mpu	maku1278	-64.3201	-11.8701	20094,21187,164199,86777,301424,braga2005aspects
-Matanau	Matanau	Mondé	???	mat	???	-60.480	-6.0372	
+Matanau	Matanau	Mondé		mat		-60.48	-6.0372	
 Mbya	Mbya	Tupi-Guarani	gun	gun	mbya1239	-52.711	-26.0188	575915,dietrich1994mbya,dooley1998lexico
 Mekens	Mekens	Tuparí	skf	skf	saki1248	-62.0927	-13.054	552771,153217,86777,hanke1958notas
 Monde	Monde	Mondé	mnd	mnd	sala1273	-61.25	-12.0499	114878,moore2005classificaccao,felzke2017dancca,galucio2015genealogical
-Mundurukú	Mundurukú	Munduruku	myu	myu	mund1330	-57.691	-8.41095	picancco2020fonologia,crofts1971repeated,crofts2004aspectos,rodrigues1980tupi,gomes2006estudo,gomes2019particulas,104115
+Munduruku	Munduruku	Munduruku	myu	myu	mund1330	-57.691	-8.41095	picancco2020fonologia,crofts1971repeated,crofts2004aspectos,rodrigues1980tupi,gomes2006estudo,gomes2019particulas,104115
 Nheengatu	Nheengatu	Tupi-Guarani	yrl	yrl	nhen1239	-66.9646	1.04503	cruz2011fonologia,100201,81291
-Ntogapid-Itogapuk	Ntogapid-Itogapuk	Ramarana-Purubora	nto	nto	nto	-61.66003	-10.452133	loukotka1968classification,67885,147062
-Old Guaraní	Old Guaraní	Tupi-Guarani	olg	olg	olg	-54.29	-26.25	restivo1893lexicon,restivo1893escolios,dietrich1984ruiz,de1876catecismo,montoya1994arte,108708
-Omagua	Omagua	nonTupían	omg	omg	omag1248	-69.17	-3.53	43829,545850,307372,575624,131312,315979
+NtogapidItogapuk	Ntogapid-Itogapuk	Ramarana-Purubora		nto		-61.66003	-10.452133	loukotka1968classification,67885,147062
```

Comment:
@StasReichert, please see this: our ID for the language is without the dash. This means, lexibank/cldfbench cannot fail here, as we have clean IDs now. I think I even added a check to google drive.